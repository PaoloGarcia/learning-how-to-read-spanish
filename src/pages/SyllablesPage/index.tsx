import Layout from "../../components/Layout"
import "./SyllablesPage.scss"

function SyllablesPage(): JSX.Element {
    return (
        <Layout backButton title="Silabas">Syllables</Layout>
    )
}

export default SyllablesPage
