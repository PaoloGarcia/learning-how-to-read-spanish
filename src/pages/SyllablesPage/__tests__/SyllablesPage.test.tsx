import React from "react"
import { render } from "@testing-library/react"
import SyllablesPage from ".."

describe("Snapshot testing", () => {
    it("renders component with no args", () => {
        const { asFragment } = render(<SyllablesPage />)
        const firstRender: DocumentFragment = asFragment()
        expect(firstRender).toMatchSnapshot()
    })
})
