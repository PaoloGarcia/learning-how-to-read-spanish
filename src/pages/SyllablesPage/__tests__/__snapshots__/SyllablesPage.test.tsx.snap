// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot testing renders component with no args 1`] = `
<DocumentFragment>
  <div
    class="Layout"
  >
    <nav
      class="back-btn"
    >
      <button
        class="ant-btn ant-btn-primary ant-btn-circle ant-btn-icon-only"
        type="button"
      >
        <span
          aria-label="arrow-left"
          class="anticon anticon-arrow-left"
          role="img"
        >
          <svg
            aria-hidden="true"
            data-icon="arrow-left"
            fill="currentColor"
            focusable="false"
            height="1em"
            viewBox="64 64 896 896"
            width="1em"
          >
            <path
              d="M872 474H286.9l350.2-304c5.6-4.9 2.2-14-5.2-14h-88.5c-3.9 0-7.6 1.4-10.5 3.9L155 487.8a31.96 31.96 0 000 48.3L535.1 866c1.5 1.3 3.3 2 5.2 2h91.5c7.4 0 10.8-9.2 5.2-14L286.9 550H872c4.4 0 8-3.6 8-8v-60c0-4.4-3.6-8-8-8z"
            />
          </svg>
        </span>
      </button>
    </nav>
    <header>
      <h1>
        Silabas
      </h1>
    </header>
    <main>
      Syllables
    </main>
  </div>
</DocumentFragment>
`;
